Lesson31/Kadir/my-app/src/components/List/index.jsx

```diff
@@ -1,34 +1,51 @@
-import {ListItem} from '../ListItem';
-import {useState} from "react";
-import './styles.css';
+import React, { useState } from "react";
+import { ListItem } from "../ListItem";
+import { AddButton } from "../AddButton";
+import "./styles.css";
 
 export const List = () => {
-  const [data, setData]  = useState([
-    {id: 1, title: "Do dishes", completed: false},
-    {id: 2, title: "Do homework", completed: true},
-    {id: 3, title: "Go running", completed: false},
-    {id: 4, title: "Do dishes 2", completed: false},
-    {id: 5, title: "Do homework 3", completed: false},
-    {id: 6, title: "Go running 4", completed: false},
+  const [data, setData] = useState([
+    { id: 1, title: "Do dishes", completed: false },
+    { id: 2, title: "Do homework", completed: true },
+    { id: 3, title: "Go running", completed: false },
+    { id: 4, title: "Do dishes 2", completed: false },
+    { id: 5, title: "Do homework 3", completed: false },
+    { id: 6, title: "Go running 4", completed: false },
   ]);
 
   const handleClick = (id) => {
-    setData((prevData) => {
-      return prevData.map(dataItem => 
-        dataItem.id === id ? {...dataItem, completed: !dataItem.completed} : dataItem)
-    });
+    setData((prevData) =>
+      prevData.map((dataItem) =>
+        dataItem.id === id
+          ? { ...dataItem, completed: !dataItem.completed }
+          : dataItem
+      )
+    );
   };
-  
-return (
-  <ul className="todo-list">
-      {
-        data.map(listItem => 
-        <ListItem key={listItem.id}
-        id={listItem.id}
-        title={listItem.title}
-        completed={listItem.completed}
-        handleClick={handleClick}/>)
-      }
-  </ul>
-)
+
+  const handleDelete = (id) => {
+    setData((prevData) => prevData.filter((dataItem) => dataItem.id !== id));
+  };
+
+  // const handleAddItem = (newItem) => {
+  //   setData([...data, newItem]);
+  // };
+
+  return (
+    <div>
+      <ul className="todo-list">
+        {data.map((listItem) => (
+          <ListItem
+            key={listItem.id}
+            id={listItem.id}
+            title={listItem.title}
+            completed={listItem.completed}
+            handleClick={handleClick}
+            handleDelete={handleDelete}
+          />
+        ))}
+      </ul>
+      <AddButton key={data.id} setData={setData} data={data}/>
```

Comment:
Nothing but the method to updated todo items is needed here:
```suggestion
      <AddButton setData={setData} />
```